docs/release/support_matrix.md

```diff
@@ -1,7 +1,428 @@
+
+
+### Unsupported Linux Distributions
+
+With DAOS Version 2.4, CentOS 7 and RHEL 7 are no longer supported.
+Please update your DAOS servers to a supported EL8 level before updating to DAOS 2.4.
+
+DAOS also does not support
+openSUSE Tumbleweed,
+Fedora,
+CentOS Linux,
+CentOS Stream,
+Ubuntu, or
+Oracle Linux.
+
+
+## Operating Systems supported for DAOS Clients
+
+The DAOS software stack is built and supported on
+Linux for the x86\_64 architecture.
+
+In DAOS Version 2.4, the supported Linux distributions and versions for DAOS clients
+are identical to those for DAOS servers. Please refer to the previous section for details.
+
+In future DAOS releases, DAOS client support may be added for additional
+Linux distributions and/or versions.
+
+
+## Fabric Support
+
+DAOS Version 2.4 supports both OFI [libfabric](https://ofiwg.github.io/libfabric/).
+and UCF [UCX](https://openucx.org/) for communication in the DAOS data plane.
+This section describes the supported network providers and contains references
+to vendor-specific information for the supported networking hardware.
+
+
+### OFI libfabric
+
+With the exception of UCX for InfiniBand networks, OFI libfabric is the recommended
+networking stack for DAOS.  DAOS Version 2.4 requires at least version 1.18.0 of
+[libfabric](https://ofiwg.github.io/libfabric/).
+It is strongly recommended to use exactly the provided libfabric version
+on all DAOS servers and all DAOS clients.
+
+Links to libfabric releases on github
+(the RPM distribution of DAOS includes libfabric RPM packages with the correct version):
+
+* [libfabric 1.18.0](https://github.com/ofiwg/libfabric/releases/tag/v1.18.0) (Apr 08, 2023)
+* [libfabric 1.18.1rc1](https://github.com/ofiwg/libfabric/releases/tag/v1.18.1rc1) (release candidate)
+
+Not all libfabric core providers listed in
+[fi\_provider(7)](https://ofiwg.github.io/libfabric/main/man/fi_provider.7.html)
+are supported by DAOS. The following providers are supported:
+
+* `ofi+tcp` is supported on all networking hardware.
+  It does not use RDMA, so on an RDMA-capable network this provider typically
+  does not achieve the maximum performance of the fabric.
+* `ofi+verbs` is supported for RDMA communication over InfiniBand fabrics.
+  Note that as an alternative to libfabric, the UCX networking stack can be
+  used on InfiniBand fabrics as described in the next subsection.
+* The `ofi+cxi` provider is supported for RDMA communication over Slingshot.
+* The `ofi+opx` (Omni-Path Express) provider is enabled as a _Technology Preview_
+  for RDMA transport over Omni-Path fabrics, for testing and evaluation purposes.
+  In production environments using Omni-Path networking, please continue to use
+  the `ofi+tcp` provider until the `ofi+opx` provider is fully supported.
+
+!!! note
+    The `ofi+psm2` provider for Omni-Path fabrics has known issues
+    when used in a DAOS context, and is not supported by DAOS Version 2.4.
```

Comment:
we removed all support for `psm2` from DAOS so maybe you can say it is deprecated at this point.

Reply:
done